src/App.vue

```diff
@@ -1,25 +1,20 @@
 <template>
+  <pre class="_debug">
+    {{ $store.state.quiz.userAnswers }}
+    </pre>
```

Comment:
Ordinarily it would be important to comment this out, but as we're about to go about using it, it's okay to leave it in for now